python_lib.py

```diff
@@ -0,0 +1,657 @@
+import sys, os, ConfigParser, math, json, time, subprocess, commands, random, string, logging, logging.handlers, socket, StringIO
```

Comment:
This whole file is still in the middle of the repository, and has a vague name. This was supposed be merged with the experiment primitive file and placed in the primitives.